packages/contracts-periphery/contracts/L1/L1ERC721Bridge.sol

```diff
@@ -89,9 +87,6 @@ contract L1ERC721Bridge is Semver, CrossDomainEnabled, OwnableUpgradeable {
     function initialize(address _messenger, address _otherBridge) public initializer {
         messenger = _messenger;
         otherBridge = _otherBridge;
```

Comment:
It would be cheaper and more consistent with the [direction](https://github.com/ethereum-optimism/optimism/pull/3443/files#diff-ab7b3add6cd9fe0b049d05778d9c14f1277231fb1a4f59c9038c26c9c6a8d32eR92) of our StandardBridge to make this an immutable variable.

Reply:
Are you referring to `messenger`, `otherBridge`, or both?

Reply:
both